source/agora/common/Ensure.d

```diff
@@ -47,14 +47,49 @@ private final class FormattedException : Exception
     {
         return this.buffer[0 .. this.end];
     }
+
+    /// Overrides `Throwable.toString` because it uses `msg`...
+    public override void toString (scope void delegate(in char[]) sink) const scope
+    {
+        import core.internal.string : unsignedToTempString;
+
+        char[20] tmpBuff = void;
+
+        sink(typeid(this).name);
+        sink("@"); sink(file);
+        sink("("); sink(unsignedToTempString(line, tmpBuff)); sink(")");
+
+        if (this.end > 0)
+        {
+            sink(": "); sink(this.message());
+        }
+        if (this.info)
```

Comment:
Where is info from?

Reply:
I'm assuming `class Exception`, which this one inherited from?

Reply:
https://github.com/dlang/druntime/blob/934b4d24bc9a24d93b1d7c380d6155cc0252f9c3/src/object.d#L2283-L2310

Reply:
I was hinting for a comment above this line.

Reply:
Thanks.